src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html

```diff
@@ -39,35 +40,29 @@
   <amp-carousel width="300"
                 height="250"
                 type="slides">
```

Comment:
@carlosfuertes FYI I removed the `div` in your carousel sample as it is not needed.